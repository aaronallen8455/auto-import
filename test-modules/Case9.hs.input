module Case9 where

main :: IO ()
main = Foo
