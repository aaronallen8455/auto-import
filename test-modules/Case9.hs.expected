module Case9 where

import Foo (Foo)

main :: IO ()
main = Foo
